Sources/OrderedSet.swift

```diff
@@ -199,6 +199,25 @@ public struct OrderedSet<T: Hashable> {
         remove(sequencedContents[index].pointee)
     }
     
+    /**
+     Removes all objects that satisfy the given predicate in the ordered set.
+     - parameter    shouldBeRemoved: A closure that takes an object in the ordered set as its argument and returns a Boolean value indicating whether the object should be removed from the ordered set.
+     - returns: The objects to be removed in ordered set.
```

Comment:
This should probably say "The objects that were removed from the ordered set." as they will be removed by the time the function returns.